assets/src/data/content/writers/renderer.tsx

```diff
@@ -7,15 +7,10 @@ import { ContentWriter } from './writer';
 interface Props {
   content: RichText;
   context: WriterContext;
-  style?: React.CSSProperties;
 }
 export const HtmlContentModelRenderer: React.FC<Props> = (props) => {
   // Support content persisted when RichText had a `model` property.
   const content = (props.content as any).model ? (props.content as any).model : props.content;
 
-  return (
-    <div style={props.style}>
-      {new ContentWriter().render(props.context, content, new HtmlParser())}
-    </div>
-  );
+  return <>{new ContentWriter().render(props.context, content, new HtmlParser())}</>;
```

Comment:
Rendering content inside a react fragement instead of a div is necessary to allow inline elements to be rendered properly. e.g. `<p></p>` technically cannot contain a `<div></div>`